app-emulation/retroarch/retroarch-1.7.1.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit flag-o-matic
+
+DESCRIPTION="A sophisticated libretro frontend for emulators, game engines and media players."
+HOMEPAGE="http://retroarch.com/"
+SRC_URI="https://github.com/libretro/RetroArch/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~arm"
+IUSE="+opengl +pulseaudio -alsa -sdl -X cg"
```

Comment:
sort it alphobetically. also, is there the reason why would you __disable__ these use flags by default?

Reply:
Done. Yes, the developers recommend not using SDL unless mandatory. PulseAudio is recommended over ALSA for their software, so no need to emerge both.